### ISSUE TITLE
afterlayout event not firing

### ISSUE THREAD

**Issue by ensconced** [antvis/G6#3713]
### Describe the bug

On version 4.6.4, the afterlayout event is not firing. For example, here is a codesandbox. https://codesandbox.io/s/hardcore-shamir-126f59

In `index.js`, it does:

```
graph.on("afterlayout", () => {
  console.log("afterlayout!!");
});
```

So I expect to see "afterlayout!!" logged in the console. But it is not logged.

If the version in package.json is changed to 4.5.1, the logging is seen.

### Your Example Website or App

https://codesandbox.io/s/hardcore-shamir-126f59

### Steps to Reproduce the Bug or Issue

1. Open the browser console
1. See that nothing is logged

### Expected behavior

1. The afterlayout event should fire

### Platform

- OS: macOS
- Browser: Chrome
- Version: Chrome 102.0.5005.61 (Official Build) (arm64)

**Comment by Yanyan-Wang** [comment]
In your demo, you did not config the 'layout' for graph

**Comment by ensconced** [comment]
I see, thank you. But still, this seems to be a breaking change. Was it a bug when it was firing on previous versions, without having a `layout` in the config?

**Comment by Yanyan-Wang** [comment]
Yes I think the 'afterlayout' should be fired only when there is a layout, otherwise only 'afterrender' should be fired. But I found there is a problem that when there is no 'layout' config on the graph instance, the 'afterrender' is not fired too. That is not expected, I will fix it in the next small version.

**Comment by Yanyan-Wang** [comment]
> Yes I think the 'afterlayout' should be fired only when there is a layout, otherwise only 'afterrender' should be fired. But I found there is a problem that when there is no 'layout' config on the graph instance, the 'afterrender' is not fired too. That is not expected, I will fix it in the next small version.

fixed by v4.6.7
